examples/tutorials/nb_python/managed_rigid_object_tutorial.py

```diff
@@ -0,0 +1,560 @@
+    rgba_camera_1stperson_spec.sensor_subtype = habitat_sim.SensorSubType.PINHOLE
+    sensor_specs.append(rgba_camera_1stperson_spec)
+
+    depth_camera_1stperson_spec = habitat_sim.CameraSensorSpec()
+    depth_camera_1stperson_spec.uuid = "depth_camera_1stperson"
+    depth_camera_1stperson_spec.sensor_type = habitat_sim.SensorType.DEPTH
+    depth_camera_1stperson_spec.resolution = camera_resolution
+    depth_camera_1stperson_spec.postition = [0.0, 0.6, 0.0]
+    depth_camera_1stperson_spec.orientation = [0.0, 0.0, 0.0]
+    depth_camera_1stperson_spec.sensor_subtype = habitat_sim.SensorSubType.PINHOLE
+    sensor_specs.append(depth_camera_1stperson_spec)
+
+    rgba_camera_3rdperson_spec = habitat_sim.CameraSensorSpec()
+    rgba_camera_3rdperson_spec.uuid = "rgba_camera_3rdperson"
+    rgba_camera_3rdperson_spec.sensor_type = habitat_sim.SensorType.COLOR
+    rgba_camera_3rdperson_spec.resolution = camera_resolution
+    rgba_camera_3rdperson_spec.postition = [0.0, 1.0, 0.3]
+    rgba_camera_3rdperson_spec.orientation = [-45, 0.0, 0.0]
+    rgba_camera_3rdperson_spec.sensor_subtype = habitat_sim.SensorSubType.PINHOLE
+    sensor_specs.append(rgba_camera_3rdperson_spec)
+
+    # agent configuration
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def simulate(sim, dt=1.0, get_frames=True):
+    # simulate dt seconds at 60Hz to the nearest fixed timestep
+    print("Simulating " + str(dt) + " world seconds.")
+    observations = []
+    start_time = sim.get_world_time()
+    while sim.get_world_time() < start_time + dt:
+        sim.step_physics(1.0 / 60.0)
+        if get_frames:
+            observations.append(sim.get_sensor_observations())
+
+    return observations
+
+
+# [/setup]
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--no-show-video", dest="show_video", action="store_false")
+    parser.add_argument("--no-make-video", dest="make_video", action="store_false")
+    parser.set_defaults(show_video=True, make_video=True)
+    args, _ = parser.parse_known_args()
+    show_video = args.show_video
+    make_video = args.make_video
+    if make_video and not os.path.exists(output_path):
+        os.mkdir(output_path)
+
+    # %%
+    # [initialize]
+    # create the simulators AND resets the simulator
+
+    cfg = make_configuration()
+    try:  # Got to make initialization idiot proof
+        sim.close()
+    except NameError:
+        pass
+    sim = habitat_sim.Simulator(cfg)
+    agent_transform = place_agent(sim)
+
+    # get the primitive assets attributes manager
+    prim_templates_mgr = sim.get_asset_template_manager()
+
+    # get the physics object attributes manager
+    obj_templates_mgr = sim.get_object_template_manager()
+
+    # get the rigid object manager
+    rigid_obj_mgr = sim.get_rigid_object_manager()
```

Comment:
Based on naming, it seems like the plan is to have a different manager for rigid objects and articulated objects?  If possible having a single manager for both would be nice.  I'm not sure if this is feasible tho

Reply:
@aclegg3 and I have been discussing how we can sync the underlying rigid object/articulated object functional overlap, so this may be realizable, but from what I know now, it will probably not be possible in the first pass of the Managed AO wrapper functionality.

Reply:
Both types of objects already share the same uuid pool, but will have different wrapper class types and interfaces. I agree it would be great if these interfaces overlap as much as possible and to have a unified "object_manager". I think this is possible and agree it would be ideal. Perhaps this will come with AO merge, but may need to be a follow-up. Either way it should be a minimal refactor since it only affects create/get/remove of objects and not manipulation.